### DISCUSSION TITLE
Implement commenting feature for the blog post

### DISCUSSION THREAD

**Discussion by hunght** [shuding/nextra#1526] [Q&A]
I want to add feature for user to comment on posts. How can I do that as I known we need to have something similar to posts/[slug].ts

**Comment by fuma-nama** [comment]
It might be hard to setup everything from zero 
I recommend using [giscus](https://giscus.app/) to integrate GitHub discussion into your blog

**Comment by yakiisama** [comment]
you can use cusdis like this or custom whatever commets you like

```jsx
const Cusdis = dynamic(() => import("nextra-theme-blog/cusdis"));

export default {
  cusdis: {
    appId: '[appId]',
  },
  comments: <Cusdis />
}
```
